### PR TITLE
Some useful changes for packaging

### PR DESCRIPTION
**Pull request by M0Rf30**
- Added binary packages generation for
  - Arch Linux
  - Fedora 31
  - openSUSE Tumbleweed
  - Ubuntu 19.10
- Added some details on prebuilt binaries usage
- Travis use latest Ubuntu Bionic agent
- Travis now upload artifacts and sha256sums 
- Travis now build new .pkg.tar.zst Arch Linux packages (+1300% in decompression and installation phase). New extension will be available on next commits.
- Gradle binary is not downloaded during Arch Linux packaging, cause it's in official repos

### CODE REVIEW

File: .travis.yml
Comment (Polpetta) [inline .travis.yml]: I'd push binaries only when we create tags, so that a tag corresponds to an official release!

That said, I'd change `prelease` from `true` for `false`. If the user really wants to try the latest version he can install AUR and use the `jkk-git` package :wink: :joy:

**Review by Polpetta — COMMENTED** [review]
Everything looks good, I commented a single thing that I'm not sure of, but after that we can merge it :+1:

**Comment by Polpetta** [comment]
This PR closes #17 once merged btw

File: .travis.yml
Comment (M0Rf30) [reply .travis.yml]: Done. Waiting for a sparkling release tag :dancer:

**Review by Polpetta — APPROVED** [review]
LGTM :+1:

**Comment by Polpetta** [comment]
@M0Rf30 the latest changes to the docker image with pacur broke something :confused:  Perhaps we should use tags to be sure to use a stable version every time

**Comment by M0Rf30** [comment]
@Polpetta yes I agree with you. I'm pushing v0.5 tag. This PR can wait

**Pull request merged**
